derive/src/decode.rs

```diff
@@ -19,7 +19,13 @@ pub fn derive(input: DeriveInput) -> TokenStream {
     let dec_impl = match dec.data.as_ref() {
         darling::ast::Data::Enum(variants) => derive_enum(&dec, variants),
         darling::ast::Data::Struct(fields) => {
-            let inner = derive_struct(&dec.ident, dec.transparent.is_some(), dec.as_array.is_some(), fields, quote!(Self));
+            let inner = derive_struct(
+                &dec.ident,
+                dec.transparent.is_some(),
+                dec.as_array.is_some(),
+                fields,
+                quote!(Self),
+            );
```

Comment:
seems like rustfmt didn't run or something?

Reply:
Hm, weird it should be in CI?